portfolio/src/main/webapp/index.html

```diff
@@ -3,16 +3,90 @@
   <head>
     <meta charset="UTF-8">
     <title>My Portfolio</title>
+    <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.5.0/css/bootstrap.min.css" integrity="sha384-9aIt2nRpC12Uk9gS9baDl411NQApFmC26EwAOH8WgZl5MYYxFfc+NcPb1dKGj7Sk" crossorigin="anonymous">
     <link rel="stylesheet" href="style.css">
+    <link href="https://fonts.googleapis.com/css2?family=Dancing+Script:wght@600&display=swap" rel="stylesheet">
     <script src="script.js"></script>
   </head>
   <body>
     <div id="content">
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
-      <p>Click here to get a random message:</p>
-      <button onclick="addRandomGreeting()">Hello</button>
-      <div id="greeting-container"></div>
+        <div class="container" id="header">
+            <div class="row">
+                <div class="col-4">
+                    <img src="imgs/profile.jpg" alt="portfolio image" id="profile-img" class="img-fluid">
+                </div>
+                <div class="col">
+                    <h1 id="title">Photography</h1>
+                    <p>I love traveling to scenic places and taking pictures on my iPhone.</p>
+                    <a href="https://cindyzhang977.github.io/cindyxzhang/">More about me</a>
+                </div>
+            </div>
+        </div>
+
+        <div class="container" id="gallery">
+            <div class="container">
+                <div class="row">
+                    <div class="dropdown">
+                        <button class="btn btn-secondary dropdown-toggle" type="button" id="dropdownMenuButton" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
```

Comment:
While optional consider the line length of your HTML and formatting it via one of the recommended approaches:
https://google.github.io/styleguide/htmlcssguide.html#HTML_Line-Wrapping